Sources/Core/Extensions/Foundation/Encodable+PovioKit.swift

```diff
@@ -15,30 +15,17 @@ public extension Encodable {
   ///
   /// do {
   ///   let encoder = JSONEncoder()
-  ///   let requestParameters = try request.encode(with: encoder)
+  ///   let requestParameters = try request.toJSON(with: encoder)
   /// } catch {
   ///   // error
   /// }
   /// ```
-  func encode(with encoder: JSONEncoder) throws -> [String: Any] {
-    do {
-      let data = try encoder.encode(self)
-      guard let jsonObject = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any] else {
-        throw JSONError.serialization
-      }
-      return jsonObject
-    } catch let encodingError as EncodingError {
-      switch encodingError {
-      case .invalidValue(_, let context):
-        Logger.debug("Encoding (failure): \(type(of: self))", params: ["error": context.debugDescription])
-      @unknown default:
-        Logger.debug("Encoding (failure): \(type(of: self))", params: ["error": "Unknown encoding error."])
-      }
-      throw encodingError
-    } catch {
-      Logger.debug("Encoding (failure): \(type(of: self))", params: ["error": error.self])
-      throw error
```

Comment:
Why removed?

Reply:
Same as above.

Reply:
Same here. Either revert the change or remove the method completely.